tests/test_quarter_adjust.py

```diff
@@ -16,64 +16,78 @@ class TestQuarterAdjust(object):
     Test whether timepoints are adjusted correctly when calling
     set_quarter_duration with the option adjust_times=True.
     """
+
     cases = [
-        dict(q_times=[0],
-             q_durs=[10],
-             note_ons=[0, 10, 20],
-             note_offs=[10, 20, 30],
-             new_q_time=[0],
-             new_q_dur=[100],
-             new_note_ons=[0, 100, 200],
-             new_note_offs=[100, 200, 300]),
-        dict(q_times=[0, 10],
-             q_durs=[10, 100],
-             note_ons=[0, 10, 110],
-             note_offs=[10, 110, 210],
-             new_q_time=[5, 10],
-             new_q_dur=[20, 200],
-             new_note_ons=[0, 15, 215],
-             new_note_offs=[15, 215, 415],
-             ),
-        dict(q_times=[0, 10],
-             q_durs=[10, 100],
-             note_ons=[0, 10, 110],
-             note_offs=[10, 110, 210],
-             new_q_time=[300, 1000],
-             new_q_dur=[1000, 2000],
-             new_note_ons=[0, 10, 110],
-             new_note_offs=[10, 110, 210],
-             ),
-        dict(q_times=[0, 10],
-             q_durs=[10, 100],
-             note_ons=[0, 10, 110],
-             note_offs=[10, 110, 210],
-             new_q_time=[5, 10],
-             new_q_dur=[20, 200],
-             new_note_ons=[0, 15, 215],
-             new_note_offs=[15, 215, 415],
-             )
+        dict(
+            q_times=[0],
+            q_durs=[10],
+            note_ons=[0, 10, 20],
+            note_offs=[10, 20, 30],
+            new_q_time=[0],
+            new_q_dur=[100],
+            new_note_ons=[0, 100, 200],
+            new_note_offs=[100, 200, 300],
+        ),
+        dict(
+            q_times=[0, 10],
+            q_durs=[10, 100],
+            note_ons=[0, 10, 110],
+            note_offs=[10, 110, 210],
+            new_q_time=[5, 10],
+            new_q_dur=[20, 200],
+            new_note_ons=[0, 15, 215],
+            new_note_offs=[15, 215, 415],
+        ),
+        dict(
+            q_times=[0, 10],
+            q_durs=[10, 100],
+            note_ons=[0, 10, 110],
+            note_offs=[10, 110, 210],
+            new_q_time=[300, 1000],
+            new_q_dur=[1000, 2000],
+            new_note_ons=[0, 10, 110],
+            new_note_offs=[10, 110, 210],
+        ),
+        dict(
+            q_times=[0, 10],
+            q_durs=[10, 100],
+            note_ons=[0, 10, 110],
+            note_offs=[10, 110, 210],
+            new_q_time=[5, 10],
+            new_q_dur=[20, 200],
+            new_note_ons=[0, 15, 215],
+            new_note_offs=[15, 215, 415],
+        ),
     ]
-    def do_test(self, q_times, q_durs,
```

Comment:
As in similar comments, I slightly prefer the old formatting of definitions, but I am ok with the new one if that is preferred by PEP8.